src/main/resources/META-INF/mods.toml

```diff
@@ -20,3 +20,14 @@ Stays true to the core of Balkon's Weapons with new item textures made by Brass_
     versionRange="[1.16.5,1.17)"
     ordering="NONE"
     side="BOTH"
+[[dependencies.brass_armory]]	
```

Comment:
This entire entry is not necessary since the loader version is already required at the top of the file, it doesn't need to be marked as a dependency (despite being shown in the Forge example)

Reply:
Oh, good to know lol